packages/home_widget/android/src/main/kotlin/es/antonborri/home_widget/HomeWidgetIntent.kt

```diff
@@ -33,8 +33,6 @@ object HomeWidgetLaunchIntent {
     }
 
     val options = ActivityOptions.makeBasic()
-    options.pendingIntentBackgroundActivityStartMode =
-        ActivityOptions.MODE_BACKGROUND_ACTIVITY_START_ALLOWED
```

Comment:
Can you add an if statement around this so this is only executed on Android Version <15